features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/utils/DrawableUtils.kt

```diff
@@ -59,8 +58,8 @@ internal class DrawableUtils(
             resizeBitmapCallback = object :
                 ResizeBitmapCallback {
                 override fun onSuccess(bitmap: Bitmap) {
-                    mainThreadHandler.post {
-                        @Suppress("ThreadSafety") // this runs on the main thread
+                    executorService.submitSafe("drawOnCanvas", internalLogger) {
```

Comment:
is it safe to operate with bitmap/canvas from the worker thread on all Android versions?

Reply:
From the tests I've done, it seems so. I'll do more test to ensure it works on all levels.

Reply:
It does work on all versions 👍